src/Taskbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+class Taskbar extends React.Component {
+  clearMarkers() {
```

Comment:
make this an arrow function as with onClick the "this" property changes this doesn't throws any error for now but we will definitely be using the this property in these functions.